gdelt/base.py

```diff
@@ -75,10 +74,12 @@ class NoDaemonProcessPool(multiprocessing.pool.Pool):
 try:
 
     codes = pd.read_json(os.path.join(BASE_DIR, 'data', 'cameoCodes.json'),
-                         dtype=dict(cameoCode='str', GoldsteinScale=np.float64))
+                         dtype=dict(cameoCode='str', GoldsteinScale=float),precise_float=True,convert_dates=False)
     codes.set_index('cameoCode', drop=False, inplace=True)
 
+
 except:  # pragma: no cover
+    print('here')
```

Comment:
Would be good to remove this print statement. It's being ran when using gdelt==0.1.14